VagrantProvisionCentOS7.sh

```diff
@@ -338,13 +338,16 @@ rm -rf ~/tmp
 echo "### Installing node-export-server..."
 sudo cp $HOOT_HOME/node-export-server/systemd/node-export.service /etc/systemd/system/node-export.service
 sudo sed -i "s|SERVICE_USER|$VMUSER|g" /etc/systemd/system/node-export.service
-sudo sed -i "s|HOOT_HOME|$HOOT_HOME|g" /etc/systemd/system/node-export.service
+sudo sed -i "s|HOOTENANNY_HOME|$HOOT_HOME|g" /etc/systemd/system/node-export.service
```

Comment:
Just curious what the thinking was to change from HOOT_HOME to HOOTENANNY_HOME.

Reply:
If I leave it as `HOOT_HOME` then the unit file would have:
```
Environment='HOOT_HOME=HOOT_HOME'
```
and the `sed` expression would break because it would replace the variable name too.

Reply:
:+1:

Reply:
:+1: